Module-10/Task1.JSONSerialization/Department.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Text;
+using System.Text.Json.Serialization;
+
+namespace Task1.JSONSerialization;
+
+public record Department
+{
+    [JsonPropertyName("Name")]
```

Comment:
Generally serialized property names start with lowercase ("name" instead of "Name")